test/e2e/start_test.go

```diff
@@ -108,7 +109,7 @@ var _ = Describe("Podman start", func() {
 		start := podmanTest.Podman([]string{"start", "-l"})
 		start.WaitWithDefaultTimeout()
 		Expect(start.ExitCode()).To(Not(Equal(0)))
-
+		time.Sleep(2 * time.Second)
```

Comment:
Why not do a 
 		wait := podmanTest.Podman([]string{"wait", "-l"})

Reply:
we are just testing a theory here.